src/server/handlers/consents.ts

```diff
@@ -48,18 +49,19 @@ export async function post (
   _context: Context,
   request: Request,
   h: ResponseToolkit): Promise<ResponseObject> {
+  // TODO: is this validation required? participantId removed from payload
   // Validate request
-  if (!isPostConsentRequestValid(request)) {
-    return h.response().code(Enum.Http.ReturnCodes.BADREQUEST.CODE)
-  }
+  // if (!isPostConsentRequestValid(request)) {
+  //   return h.response().code(Enum.Http.ReturnCodes.BADREQUEST.CODE)
+  // }
   // Asynchronously deals with creation and storing of consents and scope
   setImmediate(async (): Promise<void> => {
     try {
       await createAndStoreConsent(request)
     } catch (error) {
       logger.push(error).error('Error: Unable to create/store consent')
-      if(isMojaloopError(error)) {
-        const consentId = (request.payload as PostConsentPayload).id
+      if (isMojaloopError(error)) {
```

Comment:
I think that in case of `any error` and not only when we have `isMojaloopError` we should notify the source via `putConsentError`

Reply:
isMojaloopError: https://github.com/mojaloop/auth-service/blob/master/src/domain/errors.ts#L31

Reply:
It's existing code not sure about that.